src/bootstrap.js

```diff
@@ -123,6 +123,11 @@ export function bootstrap(initialConfig) {
 			return;
 		}
 
+		if (evolv.context) {
```

Comment:
How about warning if they do set one? Otherwise we'll be potentially spamming a lot of general users' consoles for a case we expect to happen

Reply:
> How about warning if they do set one?

I don't follow. The if-block above bails if `lazyUid` is falsy. Plus this function already indirectly results in another log message if there are more than one instance of EvolvClient on the page.